dianna/methods/kernelshap.py

```diff
@@ -175,7 +173,7 @@ def _mask_image(
         """
         # check the background color
         if background is None:
-            background = image.mean((0, 1))
+            background = image.mean(axis=(0, 1))
```

Comment:
By default, the mean of an xarray object will take the first argument as labels, but here we need axis. So is the change.